spikes/phoenix_topology/priv/static/assets/home.css

```
@@ -0,0 +1,1054 @@
+*/
+
+textarea {
+  resize: vertical;
+}
+
+/*
+1. Reset the default placeholder opacity in Firefox. (https://github.com/tailwindlabs/tailwindcss/issues/3300)
+2. Set the default placeholder color to the user's configured gray 400 color.
+*/
+
+input::-moz-placeholder, textarea::-moz-placeholder {
```

Stylelint (17.12.0)

[error] Vendor-prefixed selector "::-moz-placeholder" (selector-no-vendor-prefix)

(selector-no-vendor-prefix)